cli/clippy.toml

```diff
@@ -0,0 +1 @@
+large-error-threshold = 161
```

Comment:
161 is pretty specific, do we have such common large errors that we have to raise the threshold globally?

In either case a comment in the clippy conf would benice, or better yet a targeted attribute `allow(clippy::large_Error)` (or whatever the actual name is).